galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.vtp.manager/src/main/java/dev/galasa/vtp/internal/properties/VtpAPI.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package dev.galasa.vtp.internal.properties;
+
+import dev.galasa.framework.spi.ConfigurationPropertyStoreException;
+import dev.galasa.framework.spi.cps.CpsProperties;
+import dev.galasa.vtp.manager.VtpManagerException;
+
+/**
+ * Should the VTP Manager use the VTP API to control VTP
+ * 
+ * @galasa.cps.property
+ * 
+ * @galasa.name vtp.API.enable
+ * 
+ * @galasa.description Should the VTP manager use the VTP API
```

Comment:
why is API capatalised?   we are leaving capitals for dynamic data

Reply:
good point - caught between an acronym and dynamic data